synapse/events/utils.py

```diff
@@ -250,13 +250,13 @@ def format_event_for_client_v1(d):
 
 def format_event_for_client_v2(d):
     drop_keys = (
-        "auth_events",
-        "prev_events",
-        "hashes",
-        "signatures",
-        "depth",
-        "origin",
-        "prev_state",
+        # "auth_events",
+        # "prev_events",
+        # "hashes",
+        # "signatures",
+        # "depth",
+        # "origin",
+        # "prev_state",
```

Comment:
We don't try to drop any events when serializing events so we can use this extra data in the TARDIS DAG visualization: https://github.com/matrix-org/tardis/pull/1